### PR TITLE
Align Taranis 128*64 stats to its larger sister

### LINKED ISSUE #4950: Global Persistent Timer
There's no simple way I can find to create a persistent global timer that runs whenever the transmitter is on.  Would like to be able to keep track of how long the radio runs on a charge.  Any persistent timers created using a model revert to the last time recorded with that model active so it can't be done there.  Running Companion on Windows OS, OpenTX version 2.2.0 N371.

### PR DESCRIPTION
**Pull request by 3djc**
image

This fixes #4950

### CODE REVIEW

File: radio/src/gui/128x64/view_statistics.cpp [hunk 1]
```diff
@@ -41,15 +46,38 @@ void menuStatisticsView(event_t event)
       break;
   }
 
+#if defined(PCBTARANIS)
```
Comment (schwabe) [inline radio/src/gui/128x64/view_statistics.cpp]: Shouldn't this be CPUARM? Or do the other ARM radio not have a globalTimer?

Comment (3djc) [reply radio/src/gui/128x64/view_statistics.cpp]: That and number of timers I'm also unsure of

Comment (schwabe) [reply radio/src/gui/128x64/view_statistics.cpp]: Just checked. TIMERS defaults to 3 on all ARM platforms. You can set timers to 2 by manually compiling but that probably also breaks in Taranis/Horus world. (We might just remove that TIMER=2 option).

The only platform which is a bit strange is sky9x with reva board. There the timer might just always be zero. But I think showing a zero timer isn't problematic there. 

I would rather have all arm platforms the same here.

**Comment by bsongis** [comment]
👍

**Pull request merged**